Text/LaTeX/Base/Parser.hs

```diff
@@ -132,7 +133,7 @@ latexBlockParser = foldr1 (<|>)
 text :: Parser LaTeX
 text = do
   mbC <- peekChar
-  let nottext :: [Char]
+  let nottext :: String
```

Comment:
`[Char]` (or even `Set Char`) expresses the intend better.

Reply:
Agreed, perhaps it is better to move `nottext` out of the `text` function, and use a `Set` instead. Updated.

Reply:
Yeah. `Data.Set` is kind of overkill, but whatever.